chain/chain/src/chain.rs

```diff
@@ -523,6 +523,7 @@ impl Chain {
         }
 
         chain_store_update.save_block(block.clone());
+        chain_store_update.inc_block_refcount(&block.header.prev_hash)?;
```

Comment:
Is there a place where we'd call `save_block`, but not `inc_block_refcount`?
If no, shall we move the `inc_block_refcount` inside `save_block`?

Reply:
There are two `save_block`:
1. In Chain `inc_block_refcount` is already added to `Chain::save_block`.
2. In ChainStoreUpdate most `save_` methods update exact field stored inside ChainStoreUpdate. I'd prefer to keep control on what's updated and not messy with design of `save_` methods.